### PR TITLE
feat : 85zrwa8w3 - Add verify contract script to deploy script

### PR DESCRIPTION
**Pull request by 0xmainnov**

### CODE REVIEW

File: v2.1-contract/deploy/deploy.ts [hunk 1]
```diff
@@ -14,9 +15,9 @@ async function deploy() {
 
     // Deploy base contracts
     const authority = await HedgepieAuthority.deploy(
-        "0xB34b18b191a2371359762429f9732F73af8ac211",
-        "0xB34b18b191a2371359762429f9732F73af8ac211",
-        "0xB34b18b191a2371359762429f9732F73af8ac211"
+        process.env.GOVERNOR || "",
+        process.env.GOVERNOR || "",
+        process.env.GOVERNOR || ""
```
Comment (daiki-iu) [inline v2.1-contract/deploy/deploy.ts:20]: Use env.PATH_MANAGER and env.ADAPTER_MANAGER

File: v2.1-contract/deploy/constant.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-export const lib = { address: "0x570aB366073bBA951Dc75788a991017cFe426c23" };
+export const lib = { address: process.env.LIB };
```
Comment (daiki-iu) [inline v2.1-contract/deploy/constant.ts:1]: It does not make sense to read lib address from .env

**Review by daiki-iu — CHANGES_REQUESTED** [review]

File: v2.1-contract/deploy/constant.ts (hunk 2 above)
Comment (0xmainnov) [reply v2.1-contract/deploy/constant.ts]: Fixed, plz check again

File: v2.1-contract/deploy/deploy.ts [hunk 3]
```diff
@@ -55,6 +58,50 @@ async function deploy() {
...
     await authority.setPathFinder(pathFinder.address);
+
+    // verify base contracts
+    await verify({
+        contractName: "HedgepieAuthority",
+        address: authority.address,
+        constructorArguments: [
+            process.env.GOVERNOR || "",
+            process.env.GOVERNOR || "",
+            process.env.GOVERNOR || "",
```
[the remark this answers is not in the document]
Comment (0xmainnov) [reply v2.1-contract/deploy/deploy.ts]: Fixed, plz check again

**Review by daiki-iu — APPROVED** [review]
@0xmainnov  LGTM!

**Pull request merged**